components/brave_wallet/browser/polkadot/polkadot_keyring.h

```diff
@@ -9,10 +9,12 @@
 #include "base/containers/flat_map.h"
 #include "brave/components/brave_wallet/browser/internal/hd_key_sr25519.h"
 #include "brave/components/brave_wallet/common/brave_wallet.mojom.h"
+#include "brave/vendor/bat-native-tweetnacl/tweetnacl.h"
```

Comment:
Let's not put this dependency in header. 
We need to define kPolkadotNonceSize in this file, and static_assert for equality with crypto_secretbox_NONCEBYTES in .cc